front/app/services/__mocks__/phases.ts

```diff
@@ -117,23 +127,27 @@ export const mockPhaseSurveyTypeformData = {
   id: 'MockPhaseSurveyTypeformId',
   type: 'phase',
   attributes: {
-    title_multiloc: 'A Mock Survey Typeform phase',
-    description_multiloc: 'For testing purposes',
+    title_multiloc: { en: 'A Mock Information phase' },
+    description_multiloc: { en: 'For testing purposes' },
     start_at: 'today',
     end_at: 'one week from now',
     created_at: 'yesterday',
     updated_at: 'yesterday but later',
-    participation_method: 'survey', // 'ideation' | 'information' | 'survey' | 'budgeting'; cf participationContexts.ts
+    participation_method: 'survey' as ParticipationMethod, // 'ideation' | 'information' | 'survey' | 'budgeting'; cf participationContexts.ts
+    upvoting_method: 'limited' as VotingMethod, // 'limited' | 'unlimited'
+    downvoting_method: 'limited' as VotingMethod, // 'limited' | 'unlimited'
+    input_term: 'idea' as InputTerm,
+    presentation_mode: 'card' as PresentationMode, // 'card' | 'map'
```

Comment:
Do we need type casting here? Couldn't we just do 

export const mockPhaseInformationData: **IPhaseData** = {

?